packages/react-icon-provider/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@fluentui/react-icon-provider",
+  "version": "1.0.0-beta.0",
```

Comment:
@xugao Given recent changes to versioning practices, is this still the version we want to use for this new package?

Reply:
let's version this start with `0.1.0` like other new packages. we will also need to add this to package.json:
```
  "beachball": {
    "tag": "latest",
    "disallowedChangeTypes": [
      "major",
      "prerelease"
    ]
  },
```